### PR TITLE
fixed some small issues

### PR DESCRIPTION
**Pull request by warunicorn19**
Signed-off-by: Aditya Chaterjee <speak2adi@gmail.com>

**Description**
Fixes small issues from previous merged PRs
This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit b4dc6f6bf0e3d73a8fe08370937fa8ff4cd9a6ba at: https://618bb7cdbe7d2015f5c8b102--layer5ng.netlify.app

**Comment by Mr-DG-Wick** [comment]
@warunicorn19  While you are at it, can you also modify the CONTRIBUTING.md file ? A space needs to be added here.
Screenshot (1091)_LI

File: src/sections/Community/Handbook/faq.js [hunk 1]
```diff
@@ -15,7 +16,7 @@ const Faq= () => {
       <div className="page-section">
         <Container>
           <div className="content">
-            <h2>General FAQs:</h2>
+            <h2>General FAQs:</h2><br/>
             <h4>Q: Are Layer5’s solutions open source?</h4>
             <p>A: Yes, all of Layer5 projects are licensed under Apache V2. Layer5 also offers extensions to the Meshery project in which Enterprise-centric functionality is commercially offered and supported.</p>
```
Comment (Mr-DG-Wick) [inline src/sections/Community/Handbook/faq.js:21]:
```suggestion
            <p>A: Yes, all Layer5 projects are licensed under Apache V2. Layer5 also offers extensions to the Meshery project in which Enterprise-centric functionality is commercially offered and supported.</p>
```

**Comment by l5io** [comment]
🚀 Preview for commit e638c4c247d7f174d0baaf07f9ccd91537855ead at: https://618ce8e8ebe0e90796229e7a--layer5ng.netlify.app

**Comment by warunicorn19** [comment]
@leecalcote this is good to merge.

**Pull request merged**